calico-enterprise_versioned_docs/version-3.15/reference/component-resources/typha/prometheus.mdx

```diff
@@ -29,31 +29,38 @@ changes a "breadcrumb". Breadcrumbs are linked together into a linked list as th
+| `typha_connections_streaming`        | Number of client connections that are actively streaming (i.e. connections that successfully completed the handshake).                                                                                                                            |
+| `typha_connections_dropped`          | Total number of connections dropped due to rebalancing.                                                                                                                                                                                           |
+| `typha_kvs_per_msg`                  | Number of KV pairs sent in each message.                                                                                                                                                                                                          |
+| `typha_log_errors`                   | Number of errors encountered while logging.                                                                                                                                                                                                       |
+| `typha_logs_dropped`                 | Number of logs dropped because the output stream was blocked.                                                                                                                                                                                     |
+| `typha_next_breadcrumb_latency_secs` | Time to retrieve next breadcrumb when already behind.                                                                                                                                                                                             |
+| `typha_ping_latency`                 | Round-trip ping/pong latency to client. Typha's protocol includes a regular ping/pong keepalive to verify that the connection is still up.                                                                                                        |
+| `typha_updates_skipped`              | Total number of updates skipped because the datastore change was not relevant. (For example, an update to a Kubernetes Pod field that {{prodname}} does not read.)                                                                                |
+| `typha_updates_total`                | Total number of updates received from the datastore.                                                                                                                                                                                              |
+| `remote_cluster_connection_status`   | Status of the remote cluster connection in federation. Represented as numeric values 0 (NotConnecting) ,1 (Connecting), 2 (InSync), 3 (ReSyncInProgress), 4 (ConfigChangeRestartRequired), 5 (ConfigInComplete). Uses `remote_cluster_name` label.|
+
+Labels can be interpreted as follows:
```

Comment:
I see how this follows from `remote_cluster_connection_status`. But I'm not sure why this merits its own table. In what way does `remote_cluster_connection_status` "use" the `remote_cluster_name` label? If describing this is important only in relation to the metric, why not leave this information in the first description cell?

Reply:
We have separate table for m[etrics in kube controller.](https://docs.tigera.io/calico-enterprise/latest/reference/component-resources/kube-controllers/prometheus#kube-controllers-specific) 
However, it is useful to have separate table only when we have lot of labels related to metrics.
For now. I will move the labels along with the metrics table and more details.

Labels are important - because in this use case this metric shows it connection status with other clusters (value -1/2/5  indicates the connection status) but the labels informs about the name of the cluster connecting to.
Screen shot in this PR https://github.com/tigera/calico-private/pull/6593